front/components/assistant/conversation/input_bar/editor/mentionSuggestion.tsx

```diff
@@ -12,31 +11,23 @@ import type {
   MentionDropdownOnKeyDown,
   MentionDropdownProps,
 } from "@app/components/assistant/conversation/input_bar/editor/types";
-import { filterMentionSuggestions } from "@app/lib/mentions/editor/suggestion";
-import type { RichMention } from "@app/types";
+import type { RichMention, WorkspaceType } from "@app/types";
+
 export const mentionPluginKey = new PluginKey("mention-suggestion");
 
-export function createMentionSuggestion() {
+export function createMentionSuggestion({
+  owner,
+  conversationId,
+}: {
+  owner: WorkspaceType;
+  conversationId: string | null;
+}) {
   return {
     pluginKey: mentionPluginKey,
     // Ensure queries can contain spaces (e.g., @Sales Team → decomposes to
     // text and keeps the dropdown active over the full label).
     allowSpaces: true,
 
-    items: ({
```

Comment:
all the fetching is done in the MentionDropdown, so we can remove this part. 

This now only does the passing between tiptap and MentionDropdown